cekit-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script builds the image locally
+
+# pnc list-built-artifacts -q 'filename=like="%.zip"' BUILD_ID | ./get-artifacts.py > artifacts-override.yaml
```

Comment:
This could be enhanced yet to take the _BUILD\_ID_, desired _JDK_ and _BUILDER_ as CLI parameters